README.md

```diff
@@ -10,34 +10,38 @@
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 **Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
 
-- [Overview](#overview)
-- [Quick Start](#quick-start)
-  - [Setting up a channel to sync resourcese between your hub cluster and a object bucket](#setting-up-a-channel-to-sync-resourcese-between-your-hub-cluster-and-a-object-bucket)
-  - [Trouble shooting](#trouble-shooting)
-- [Community, discussion, contribution, and support](#community-discussion-contribution-and-support)
-- [Security Response](#security-response)
-- [References](#references)
-  - [multicloud-operators repositories](#multicloud-operators-repositories)
+- [multicloud-operators-application](#multicloud-operators-application)
+    - [What is the multicloud-operators-application](#what-is-the-multicloud-operators-application)
+    - [Getting started](#getting-started)
+    - [References](#references)
```

Comment:
Organized it the same as app readme, but we have a procedure in this one.

Reply:
it should be `multicloud-operators-channel` for the first 2 rows